### PR TITLE
Moderated appraisal form

### PR DESCRIPTION
**Pull request by jaleszek**

### CODE REVIEW

File: app/models/assessor_assignment.rb [hunk 1]
```diff
@@ -22,6 +20,7 @@ class AssessorAssignment < ActiveRecord::Base
     end
 
     validate :submitted_at_immutability
+    validate :assessor_existence
```
Comment (matthewford) [inline app/models/assessor_assignment.rb]: Inconsistent indentation detected.

File: app/models/form_answer.rb [hunk 2]
```diff
@@ -58,13 +58,13 @@ class FormAnswer < ActiveRecord::Base
     has_many :assessors, through: :assessor_assignments do
       def primary
         where(assessor_assignments: {
-          position: AssessorAssignment::PRIMARY_POSITION
+          position: "primary"
```
Comment (matthewford) [inline app/models/form_answer.rb]: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.

File: app/models/form_answer.rb [hunk 3]
```diff
@@ -58,13 +58,13 @@ class FormAnswer < ActiveRecord::Base
...
         }).first
       end
 
       def secondary
         where(assessor_assignments: {
-          position: AssessorAssignment::SECONDARY_POSITION
+          position: "secondary"
```
Comment (matthewford) [inline app/models/form_answer.rb]: Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.

File: spec/features/assessor/appraisal_form_submission_spec.rb [hunk 4]
```diff
@@ -11,6 +11,7 @@
   let!(:form_answer) { create(:form_answer, :innovation) }
   let(:primary) { "#section-appraisal-form-primary" }
   let(:secondary) { "#section-appraisal-form-secondary" }
+  let(:moderated) { "#section-appraisal-form-moderated"}
```
Comment (matthewford) [inline spec/features/assessor/appraisal_form_submission_spec.rb]: Space missing inside }.

**Comment by RuslanHamidullin** [comment]
Good to merge :+1:

**Pull request merged**